analyzers/src/SonarAnalyzer.Common/DiagnosticAnalyzer/RuleDescriptor.cs

```diff
@@ -0,0 +1,40 @@
+﻿/*
+ * SonarAnalyzer for .NET
+ * Copyright (C) 2015-2022 SonarSource SA
+ * mailto: contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using SonarAnalyzer.Common;
+
+namespace SonarAnalyzer.Helpers
```

Comment:
Maybe we should move all the classes from this folder to `SonarAnalyzer.DiagnosticAnalyzer`, or to the `Common` folder and update the namespace.

Reply:
I've also moved `SourceScope` from `Helpers` to `Common`. It was causing too much trouble there